lib/services/workflow-api-service.js

```diff
@@ -155,7 +236,22 @@ function workflowApiServiceFactory(
     };
 
     WorkflowApiService.prototype.findActiveGraphForTarget = function(target) {
-        return taskGraphStore.findActiveGraphForTarget(target);
+        return waterline.graphobjects.findOne({
+            node: target,
+            _status: Constants.Task.ActiveStates
+        });
+    };
+
+    WorkflowApiService.prototype.getActiveWorkflows = function(query) {
```

Comment:
This API name is not well-matched with its implementation. If the API automatically builds the query to lookup active workflow, then I think the name and implementation is matched well.